sphinx/ext/hieroglyph_patch.py

```diff
@@ -0,0 +1,41 @@
+"""Monkey patch hieroglyph to work with Sphinx 1.8.0+.
+
+This extension serves as a temporary workaround, for more information see
+`https://github.com/nyergler/hieroglyph/issues/148`_.
+
+If an extension provides its own visit or depart methods, patch them into the
+``HTMLTranslator`` class below.
+
+It would be possible to patch the ``add_node`` method of the Sphinx application
+to patch extensions automatically but in the interests of keeping the hack to a
+minimum this hard-coded extension should suffice.
+
+"""
+
+import sphinx
+
+from sphinx.writers.html import HTMLTranslator
+
+from sphinx.ext.graphviz import html_visit_graphviz
+from sphinx.ext.autosummary import (autosummary_toc_visit_html,
+                                    autosummary_table_visit_html)
+from minicylc import MiniCylc
+
+
+def none(*args, **kwargs):
```

Comment:
Is this just left from development or does it have some mysterious purpose?

Reply:
It's a dummy interface used below on lines 36 and 38.

Reply:
Oh, sorry, I did check for that but missed them (can't have looked very well!)